database/buildSchema/10B_template_filter.sql

```diff
@@ -0,0 +1,17 @@
+-- template filter tables
+CREATE TABLE public.filter (
+    id serial PRIMARY KEY,
+    code varchar NOT NULL UNIQUE,
+    icon_color varchar DEFAULT 'blue',
+    icon varchar,
+    title varchar,
+    query jsonb,
+    user_role public.permission_policy_type
+);
+
+CREATE TABLE public.template_filter_join (
+    id serial PRIMARY KEY,
+    template_id integer REFERENCES public.template (id),
+    template_filter_id integer REFERENCES public.filter (id)
```

Comment:
This column should probably just be called `filter_id`

Reply:
447877af67391a8a7fa0f6d883766a8b8e999bcb